### PR TITLE
Update common-lua-scripts.rst to include docs.lua

### PR DESCRIPTION
**Pull request by timtuxworth**
When adding a method to lua bindings you have to update docs.lua manually

### CODE REVIEW

File: common/source/docs/common-lua-scripts.rst [hunk 1]
```diff
@@ -532,6 +532,7 @@ To give Lua scripts access to more features of ArduPilot the API can be extended
 
 - Find the method or function you would like to expose to Lua. For example if you wanted to expose an additional feature of AHRS you would first find the method within `libraries/AP_AHRS/AP_AHRS.h <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_AHRS/AP_AHRS.h>`__. This can be an already existing method (function) or a method (function) newly added to the code.
 - Edit the `libraries/AP_Scripting/generator/description/bindings.desc <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Scripting/generator/description/bindings.desc>`__ and add a new line in the appropriate section for the method, or add a new section if a new class shall be added by following the examples of the other sections.
+- Add the method or function to `libraries/AP_Scripting/docs/docs.lua so that your IDE can provide coding prompts such as Intellisense in VSCode
```
Comment (IamPete1) [inline common/source/docs/common-lua-scripts.rst:535]:
```suggestion
- Add the method or function to `libraries/AP_Scripting/docs/docs.lua<https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Scripting/docs/docs.lua>`__.
```
That should make it a proper link. I would remove the Intellisencse bit, its this extension. https://marketplace.visualstudio.com/items?itemName=sumneko.lua but I don't think its necessary to say what the docs are for. There is another section on how to setup the extension.

**Comment by timtuxworth** [comment]
ok @Hwurzburg ?

File: common/source/docs/common-lua-scripts.rst [hunk 2]
```diff
@@ -532,6 +532,7 @@ To give Lua scripts access to more features of ArduPilot the API can be extended
 
 - Find the method or function you would like to expose to Lua. For example if you wanted to expose an additional feature of AHRS you would first find the method within `libraries/AP_AHRS/AP_AHRS.h <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_AHRS/AP_AHRS.h>`__. This can be an already existing method (function) or a method (function) newly added to the code.
 - Edit the `libraries/AP_Scripting/generator/description/bindings.desc <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Scripting/generator/description/bindings.desc>`__ and add a new line in the appropriate section for the method, or add a new section if a new class shall be added by following the examples of the other sections.
+- Add the method or function to `libraries/AP_Scripting/docs/docs.lua<https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Scripting/docs/docs.lua>`__.
```
Comment (Hwurzburg) [inline common/source/docs/common-lua-scripts.rst:535]: this is a build error
```suggestion
- Add the method or function to `libraries/AP_Scripting/docs/docs.lua <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Scripting/docs/docs.lua>`__.
```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: common/source/docs/common-lua-scripts.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-lua-scripts.rst:535]: remember to squash to one commit when done

Comment (timtuxworth) [reply common/source/docs/common-lua-scripts.rst]: Will do if you are good with how it looks now?

Comment (Hwurzburg) [reply common/source/docs/common-lua-scripts.rst]: go ahead

**Comment by timtuxworth** [comment]
Should be good now @Hwurzburg

**Pull request merged**